server/testdata/integration/config/03_couper.hcl

```diff
@@ -236,16 +236,6 @@ server "acs" {
     }
   }
 
-  endpoint "/jwks/rsa/not_found" {
-    disable_access_control = ["ba1"]
-    access_control = ["JWKS_not_found"]
-    response {
-      headers = {
-        x-jwt-sub = request.context.JWKS_not_found.sub
-      }
-    }
-  }
-
```

Comment:
Was passiert, wenn das JWKS-Backend im laufenden Betrieb ausfällt? Wo wird das jetzt getestet?

Reply:
Scheint nicht relevant zu sein, wenn man auf den Diff schaut, ein älterer BE commit?
Dann würde ggf eine vorhandene Config da sein und schlicht nicht aktualisiert.

Reply:
Ja, das gehört nicht zu "Health", war mir aber aufgefallen.

Dieser Test testete doch offenbar, was passiert, wenn ein JWKS-URI-Request einen 404 (oder Fehler allgemein) produziert. Das kann ja unabhängig von einem Couper-Start passieren. Bin nicht der Meinung, dass dieser Test überflüssig geworden ist.